packages/core/src/definitions/styles.ts

```diff
@@ -319,19 +319,137 @@ export const containerBuiltInStyles: Partial<
   },
 };
 
-export const singleColumnBuiltInStyles = {
-  ...builtInStyles,
-  cfColumnSpan: {
+export const singleColumnBuiltInStyles: Partial<
+  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Boolean'>>
+> = {
+  cfVerticalAlignment: {
+    validations: {
+      in: [
+        {
+          value: 'start',
+          displayName: 'Align left',
+        },
+        {
+          value: 'center',
+          displayName: 'Align center',
+        },
+        {
+          value: 'end',
+          displayName: 'Align right',
+        },
+      ],
+    },
     type: 'Text',
-    defaultValue: '6',
     group: 'style',
-  } as ComponentDefinitionVariable<'Text'>,
-  cfHeight: {
-    displayName: 'Height',
+    description: 'The horizontal alignment of the section',
+    defaultValue: 'center',
+    displayName: 'Vertical alignment',
+  },
+  cfHorizontalAlignment: {
+    validations: {
+      in: [
+        {
+          value: 'start',
+          displayName: 'Align top',
+        },
+        {
+          value: 'center',
+          displayName: 'Align center',
+        },
+        {
+          value: 'end',
+          displayName: 'Align bottom',
+        },
+      ],
+    },
     type: 'Text',
     group: 'style',
-    description: 'The height of the section',
-    defaultValue: 'auto',
+    description: 'The horizontal alignment of the section',
+    defaultValue: 'center',
+    displayName: 'Horizontal alignment',
+  },
+  cfPadding: {
+    displayName: 'Padding',
+    type: 'Text',
+    group: 'style',
+    description: 'The padding of the section',
+    defaultValue: '0 0 0 0',
+  },
+  cfBackgroundColor: {
+    displayName: 'Background',
+    type: 'Text',
+    group: 'style',
+    description: 'The background color of the section',
+    defaultValue: 'rgba(255, 255, 255, 0)',
+  },
+  cfFlexDirection: {
+    displayName: 'Direction',
+    type: 'Text',
+    group: 'style',
+    description: 'The orientation of the section',
+    defaultValue: 'column',
+  },
+  cfFlexWrap: {
+    displayName: 'Wrap objects',
+    type: 'Text',
+    group: 'style',
+    description: 'Wrap objects',
+    defaultValue: 'nowrap',
+  },
+  cfBorder: {
+    displayName: 'Border',
+    type: 'Text',
+    group: 'style',
+    description: 'The border of the section',
+    defaultValue: '0px outside rgba(255, 255, 255, 1)',
```

Comment:
Isn't this just a white border? Should we just make it transparent like the `cfBackgroundColor`?